src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/components/commerce/productteaser/productteaser.html

```diff
@@ -0,0 +1,61 @@
+<!--/*
+    Copyright 2019 Adobe Systems Incorporated
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/-->
+
+<sly
+  data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html"
+  data-sly-use.product="${package}.core.models.MyProductTeaser"
+>
+  <sly data-sly-test.ispropavailable="${symbol_dollar}{properties.selection}">
+    <div data-sly-test.isvalid="${symbol_dollar}{product.url}" class="item__root">
+      <div data-sly-test="${symbol_dollar}{product.showBadge}" class="item__badge">
+        <span>${symbol_dollar}{properties.text || 'New'}</span>
+      </div>
+      <a class="item__images" href="${symbol_dollar}{product.url}">
+        <img
+          class="item__image"
+          width="100%"
+          height="100%"
+          src="${symbol_dollar}{product.image}"
+          alt="${symbol_dollar}{product.image}"
+        />
+      </a>
+      <a class="item__name" href="${symbol_dollar}{product.url}"
+        ><span>${symbol_dollar}{product.name}</span></a
+      >
+      <div class="item__price">
+        <span> ${symbol_dollar}{product.formattedPrice} </span>
+      </div>
+    </div>
+    <!-- Condition if the product is not available, or invalid sku provided -->
+    <div class="blank-placeholder" data-sly-test="${symbol_dollar}{!isvalid}">
+      <strong
```

Comment:
https://github.com/adobe/aem-core-cif-components/pull/162 introduces `i18n` so please make sure to also do this here.

Reply:
Incorporated that now as well.